src/tournament-mode/remote-peer-menu.tsx

```diff
@@ -19,12 +27,26 @@ export function RemotePeerControls() {
   }
 
   let coreControl: JSX.Element;
+  // Remove outline from link icon
+  let removeOutline = {backgroundColor: 'transparent', boxShadow: 'none', outline: 'none'};
```

Comment:
I think this can mostly be accomplished with the `minimal` prop on the Blueprint button component.